packages/alfa-rules/test/common/predicate/is-visible.spec.tsx

```diff
@@ -61,6 +61,37 @@ test(`isVisible() returns false when an element is hidden by reducing its size
   t.equal(isVisible(element), false);
 });
 
+test(`isVisible() returns false when an element is hidden by reducing its size
+      to 1x1 pixels and clipping overflow`, (t) => {
+  const element = (
+    <div style={{ width: "0px", height: "0px", overflow: "hidden" }}>
+      Hello World
+    </div>
+  );
+
+  t.equal(isVisible(element), false);
+});
+
+test(`isVisible() returns false when an element is scroll and its size is 0x0 px`, (t) => {
+  const element = (
+    <div style={{ width: "0px", height: "0px", overflow: "scroll" }}>
```

Comment:
That should probably be split in several cases for each dimension just like https://github.com/Siteimprove/alfa/blob/develop/packages/alfa-rules/test/common/predicate/is-visible.spec.tsx#L34-L51 to show that individual dimensions of 0 are enough (it's not about having **both** height and width at 0)

(same for the `auto` case)